digitalpublicgoods/treejer-protocol.json

```diff
@@ -74,9 +84,9 @@
       "stepsAddressRiskPreventSafetyUnderageUsers": [
         ""
       ],
-      "griefAbuseHarassmentProtection": "Yes",
+      "griefAbuseHarassmentProtection": "",
       "harassmentProtectionSteps": [
-        "https://docs.treejer.com/project-charter"
```

Comment:
Should this link be changed out to https://docs.treejer.com/project-charter#da-contributor-covenant-code-of-conduct ?

Reply:
Resolved in commit e34da24